run/README.md

```diff
@@ -127,7 +128,8 @@ gcloud beta run deploy $SAMPLE \
 [helloworld]: https://github.com/knative/docs/tree/master/docs/serving/samples/hello-world/helloworld-java
 [pubsub]: pubsub/
 [run_button_helloworld]: https://console.cloud.google.com/cloudshell/editor?shellonly=true&cloudshell_image=gcr.io/cloudrun/button&cloudshell_git_repo=https://github.com/knative/docs&cloudshell_working_dir=docs/serving/samples/hello-world/helloworld-java
-[run_button_pubsub]: https://console.cloud.google.com/cloudshell/editor?shellonly=true&cloudshell_image=gcr.io/cloudrun/button&cloudshell_git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&cloudshell_working_dir=run/cloudrun-pubsub
+[run_button_pubsub]: https://console.cloud.google.com/cloudshell/editor?shellonly=true&cloudshell_image=gcr.io/cloudrun/button&cloudshell_git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&cloudshell_working_dir=run/pubsub
```

Comment:
I saw a new URL in #1620  from @jamesward today - you might wish to verify this. (and all the others)

Reply:
```suggestion
[run_button_pubsub]: https://deploy.cloud.run/?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&dir=run/pubsub
```

Reply:
done.

Reply:
Maybe a GitHub UI thing but I'm not seeing that change.  Note that you should be able to accept a suggestion from the GitHub PR UI.

Reply:
Will push the update soon!